### PR TITLE
Catalog offline mode, Add to List button fixes

### PR DESCRIPTION
**Pull request by Abhyastamita**
- Add to List now shows in 'Catalog Offline, login allowed with eContent Active' mode for modals.
- Add to List now does not show for List searches.

### CODE REVIEW

**Comment by Abhyastamita** [comment]
This PR now also includes an unrelated change to translate the error message from OverDrive for trying to return a Kindle book.

File: code/web/Drivers/OverDriveDriver.php [hunk 1]
```diff
@@ -1295,7 +1295,14 @@ public function returnCheckout($patron, $overDriveId) {
 				'isPublicFacing' => true,
 			]);
 			if (isset($response->message)) {
-				$cancelHoldResult['message'] .= "  {$response->message}";
+                if ($response->message == "An unmapped error has occurred. '7'") {
```
Comment (mdnoble73) [inline code/web/Drivers/OverDriveDriver.php]: Can you fix tabs vs. spaces for this?

**Review by mdnoble73 — CHANGES_REQUESTED** [review]
Just a couple of space vs. tab fixes

**Comment by Abhyastamita** [comment]
I think all the tabs are fixed now.

**Review by mdnoble73 — APPROVED** [review]

**Pull request merged**